src/main/resources/styles/reporting.css

```diff
@@ -1,5 +1,9 @@
 <style>
 
+h2 {
+	font-size: 24px !important;
```

Comment:
Because of the often long project names from jenkins the font should be reduced in size. The '!important' overrides the former declarations. May be styling can be revised.